src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -245,13 +258,6 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
         bs.setString("event_manifest", m.eventManifest)
         bs.setBytes("event", m.serialized)
 
-        if (session.protocolVersion.compareTo(ProtocolVersion.V4) < 0) {
-          bs.setToNull("message")
-          (1 to maxTagsPerEvent).foreach(tagId => bs.setToNull("tag" + tagId))
-        } else {
-          bs.unset("message")
-        }
```

Comment:
@patriknw thanks for having a look.

Regarding `message` - i have completely removed this column from the write statements, because it is indeed not needed any more.

Regarding `tags` - as far as i understand, we need to support tags for current versions of the library. But we have no idea what cassandra version are we persisting to. So i think we do need the 4 statements. Here's why: If someone is using the latest version of the library on a ` 2.1.x` cassandra, writing just 1 tag, with a `tag1, tag2, tag ... ?, ?, ?` statement, the 2 NULL tags will generate tombstones.  I would love to reduce the statement number, but i don't see how...